src/components/styles/layout.module.css

```diff
@@ -0,0 +1,33 @@
+.layoutBox {
+    margin: 3em auto;
+    margin-top: 0;
+    max-width: 650;
```

Comment:
Could be wrong but I don't think this value will do anything — in CSS, measurements (except for 0 and `auto`) usually require units (like `650px`). Something to note when working on styles.